### PR TITLE
Copy updates to My Plan (Jetpack)

### PR DESCRIPTION
**Pull request by richardmuscat**
Simplifying various text blocks.

plans_ _testing_things_ _wordpress_com

### CODE REVIEW

File: client/blocks/product-purchase-features/product-purchase-features-list/jetpack-backup-security.jsx [hunk 1]
```diff
@@ -14,13 +14,11 @@ export default localize( ( { translate } ) => {
...
-				title={ translate( 'Backups & Security' ) }
+				title={ translate( 'Site Security' ) }
 				description={ translate(
-					'VaultPress makes it easy to keep an up-to-date backup of your site with both daily and real-time syncing of all ' +
-					'your WordPress content. To ensure your site stays safe, VaultPress performs security scans daily and makes it ' +
-					'easy to review and fix threats.'
+					'Your site is being securely backed up and scanned with real-time sync.'
 				) }
-				buttonText={ translate( 'View your backups' ) }
+				buttonText={ translate( 'Visit Security Dashboard' ) }
```
Comment (a8ci18n) [inline client/blocks/product-purchase-features/product-purchase-features-list/jetpack-backup-security.jsx]: Hi! I've found a possible matching string that has already been translated **85** times:
`translate( 'Visit Dashboard' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/product-purchase-features/product-purchase-features-list/jetpack-backup-security.jsx [hunk 2]
```diff
@@ -14,13 +14,11 @@ export default localize( ( { translate } ) => {
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
 				icon="flag"
-				title={ translate( 'Backups & Security' ) }
+				title={ translate( 'Site Security' ) }
```
Comment (a8ci18n) [inline client/blocks/product-purchase-features/product-purchase-features-list/jetpack-backup-security.jsx]: :disappointed: 57 existing translations will be lost with this change.

File: client/blocks/product-purchase-features/product-purchase-features-list/jetpack-return-to-dashboard.jsx [hunk 3]
```diff
@@ -21,7 +21,7 @@ export default localize( ( { selectedSite, translate } ) => {
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
 				icon="house"
-				title={ translate( 'Return to your site\'s dashboard' ) }
+				title={ translate( 'Return to Your Dashboard' ) }
```
Comment (a8ci18n) [inline client/blocks/product-purchase-features/product-purchase-features-list/jetpack-return-to-dashboard.jsx]: Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Return to your site's dashboard' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jetpack-myplan-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/product-purchase-features/product-purchase-features-list/jetpack-wordpress-com.jsx [hunk 4]
```diff
@@ -14,17 +14,17 @@ export default localize( ( { selectedSite, translate } ) => {
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
 				icon="plugins"
-				title={ translate( 'Get the most from WordPress.com' ) }
+				title={ translate( 'Automatic Updates' ) }
```
Comment (a8ci18n) [inline client/blocks/product-purchase-features/product-purchase-features-list/jetpack-wordpress-com.jsx]: Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Enable Automatic Updates' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jetpack-myplan-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by seear** [comment]
My only query with this one, @richardmuscat, is that the button text is changing from sentence-case to title-case, and there are still some sentence-case buttons visible in the screenshot (the support buttons). Is this an intentional change?

File: client/blocks/product-purchase-features/product-purchase-features-list/jetpack-return-to-dashboard.jsx (hunk 3 above)
Comment (tyxla) [inline client/blocks/product-purchase-features/product-purchase-features-list/jetpack-return-to-dashboard.jsx]: Not sure if this one is better with title case. Personally I'd use sentence case for it.

Comment (richardmuscat) [reply client/blocks/product-purchase-features/product-purchase-features-list/jetpack-return-to-dashboard.jsx]: Fixed!

File: client/blocks/product-purchase-features/product-purchase-features-list/jetpack-return-to-dashboard.jsx [hunk 5]
```diff
@@ -21,7 +21,7 @@ export default localize( ( { selectedSite, translate } ) => {
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
 				icon="house"
-				title={ translate( 'Return to your site\'s dashboard' ) }
+				title={ translate( 'Return to your dashboard' ) }
```
Comment (a8ci18n) [inline client/blocks/product-purchase-features/product-purchase-features-list/jetpack-return-to-dashboard.jsx]: Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Return to your site's dashboard' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/jetpack-myplan-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Pull request merged**